Update changelog (#12324)
mapbox/mapbox-gl-js: CHANGELOG.md

## 2.11.0-beta.1

### Features ✨ and improvements 🏁

* Add support for `cameraForBounds` with globe projection ([#12138](https://github.com/mapbox/mapbox-gl-js/pull/12138))
* Add support for `fitBounds` and `fitScreenCoordinates` with globe projection ([#12211](https://github.com/mapbox/mapbox-gl-js/pull/12211))
* Improve support for `getBounds` with globe projection. ([#12286](https://github.com/mapbox/mapbox-gl-js/pull/12286))
* Improve symbol placement performance with globe projection ([#12105](https://github.com/mapbox/mapbox-gl-js/pull/12105))

> * Add new marker styling option `terrainOccludedMarkerOpacity` allowing the user to set the opacity of a marker that's behind 3D terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))

Reviewer: We changed the name of that marker option:
```suggestion
* Allow to override marker opacity when occluded by terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))
```

Reviewer: Just saw @SnailBones suggestion above to initially reword that, maybe let's only switch the name:
```suggestion
* Add new marker styling option `occludedOpacity` allowing the user to set the opacity of a marker that's behind 3D terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))
```

## 2.11.0-beta.1

### Features ✨ and improvements 🏁

> * Improve symbol placement performance with globe projection ([#12105](https://github.com/mapbox/mapbox-gl-js/pull/12105))

Reviewer 2: Nit: Should this go at the bottom of this section? IMO the bounds support might be the most impactful parts of this release and it would be best to list them first.

> * Add support for cameraForBounds with globe view ([#12138](https://github.com/mapbox/mapbox-gl-js/pull/12138))

Reviewer 2: ```suggestion
* Add support for `cameraForBounds` with globe projection ([#12138](https://github.com/mapbox/mapbox-gl-js/pull/12138))
```

> * Add support for fitBounds and fitScreenCoordinates with globe projection ([#12211](https://github.com/mapbox/mapbox-gl-js/pull/12211))

Reviewer 2: ```suggestion
* Add support for `fitBounds` and `fitScreenCoordinates` with globe projection ([#12211](https://github.com/mapbox/mapbox-gl-js/pull/12211))
```

> * Allow to override marker opacity when occluded by terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))

Reviewer 2: ```suggestion
* Add new marker styling option `terrainOccludedMarkerOpacity` allowing the user to set the opacity of a marker that's behind 3D terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))
```

* Improve support for `getBounds` with globe projection. ([#12286](https://github.com/mapbox/mapbox-gl-js/pull/12286))
* Cancel ImageSource image request when underlying resource is no longer used ([#12266](https://github.com/mapbox/mapbox-gl-js/pull/12266)) (h/t [maciejmatu](https://github.com/maciejmatu))

> * Add object literal support in LngLatBounds.extend ([#12270](https://github.com/mapbox/mapbox-gl-js/pull/12270)) (h/t [stampyzfanz](https://github.com/stampyzfanz))

Reviewer 2: ```suggestion
* Add object literal support in `LngLatBounds.extend` ([#12270](https://github.com/mapbox/mapbox-gl-js/pull/12270)) (h/t [stampyzfanz](https://github.com/stampyzfanz))
```

## 2.11.0-beta.1

### Features ✨ and improvements 🏁

* Improve symbol placement performance with globe projection ([#12105](https://github.com/mapbox/mapbox-gl-js/pull/12105))
* Add support for cameraForBounds with globe view ([#12138](https://github.com/mapbox/mapbox-gl-js/pull/12138))
* Add support for fitBounds and fitScreenCoordinates with globe projection ([#12211](https://github.com/mapbox/mapbox-gl-js/pull/12211))
* Allow to override marker opacity when occluded by terrain (h/t [jacadzaca](https://github.com/jacadzaca)) ([#12258](https://github.com/mapbox/mapbox-gl-js/pull/12258))

> * Improve the globe bounds calculation ([#12286](https://github.com/mapbox/mapbox-gl-js/pull/12286))

Reviewer 2: More specificity might help make it clear this is different from the above PRs for camerForBounds and fitBounds
```suggestion
* Improve support for `getBounds` with globe projection. ([#12286](https://github.com/mapbox/mapbox-gl-js/pull/12286))
```

* Fix incorrect shading of corners in fill extrusions when ambient occlusion is enabled. ([#12214](https://github.com/mapbox/mapbox-gl-js/pull/12214))
* Fix potential performance regression on image source updates ([#12212](https://github.com/mapbox/mapbox-gl-js/pull/12212))
* Fix memory leak when removing maps ([#12224](https://github.com/mapbox/mapbox-gl-js/pull/12224)) (h/t [joewoodhouse](https://github.com/joewoodhouse))
* Fix jumping when scrolling with mouse when crossing the antimeridian on projections that wrap. ([#12238](https://github.com/mapbox/mapbox-gl-js/pull/12238))
* Fix updating marker position when toggling between world copied projections and projections without ([#12242](https://github.com/mapbox/mapbox-gl-js/pull/12242))
* Fix missing ocean tiles when globe is enabled. ([#12279](https://github.com/mapbox/mapbox-gl-js/pull/12279))
* Fix missing icons in some styles. ([#12299](https://github.com/mapbox/mapbox-gl-js/pull/12299))
* Fix a bug where setting promoteId for some layers broke feature ids on other layers. ([#12322](https://github.com/mapbox/mapbox-gl-js/pull/12322)) (h/t [yongjun21](https://github.com/yongjun21))

> * Fix cursor to return to original state after a popup with `trackPointer` is removed ([#12230](https://github.com/mapbox/mapbox-gl-js/pull/12230)) (h/t [camouflagedName](https://github.com/camouflagedName))

Reviewer 2: ```suggestion
* Fix cursor returning to original state after a popup with `trackPointer` is removed ([#12230](https://github.com/mapbox/mapbox-gl-js/pull/12230)) (h/t [camouflagedName](https://github.com/camouflagedName))
```
